### ISSUE TITLE
Vertical mode not discoverable

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#160314]
I read numerous feedback that people can not find the vertical mode in merge editor. For example: https://twitter.com/ACSoko/status/1566814542646185986?s=20&t=GtES_eoS52ESMCCOADr3AQ

Opening this issue so we consider if this deserves to be a primary action and not behind `...`. The problem with the `...` menu is that there are multiple of those and they are not very discoverable.

I would also suggest we remove the "Split Editor Right" action. I think it just takes up space and it could be used by a merge editor specific action. I know @bpasero is against this. Would it help if I find numbers how many users click on this?

I am not sure if putting it in the primary action is the way to go, but would like to hear your thoughts.

fyi @daviddossett

**Comment by jrieken** [comment]
How about changing the default mode?

**Comment by daviddossett** [comment]
Agreed. This was on my list for this iteration too. I'll find an icon that would be suitable if promoted to the main toolbar.

Would be very curious on how other's feel about the vertical mode being the default. IMO that would be an easier case to make if we used the IntelliJ-style arrow buttons instead of checkboxes. But curious to hear more thoughts. 

![ws_resolveConflict](https://user-images.githubusercontent.com/25163139/188910210-fd322255-4bde-4717-936b-e69ae05b952c.png)

**Comment by daviddossett** [comment]
What about bringing back the toolbar button with the current state represented by the icon?

<img width="817" alt="CleanShot 2022-09-07 at 15 03 55@2x" src="https://user-images.githubusercontent.com/25163139/188991922-7f20d7ee-c4c7-4407-bf00-87e48467758a.png">
<img width="898" alt="CleanShot 2022-09-07 at 15 04 25@2x" src="https://user-images.githubusercontent.com/25163139/188991930-cce6e1c0-9bbc-4904-b8f8-f637a4690360.png">

**Comment by jrieken** [comment]
Also all-in for having this as a top-level submenu.

**Comment by isidorn** [comment]
Yeah! Looks great as a top-level submenu.
Let's not change the default layout yet. There are a lot of moving pieces and would like to minimise those if possible.
